extensions/khr/GL_KHR_non_const_texture_offset.txt

```diff
@@ -0,0 +1,79 @@
+Name
+
+    KHR_non_const_texture_offset
+
+Name Strings
+
+    GL_KHR_non_const_texture_offset
+
+Contact
+
+    Ricardo Garcia (rgarcia 'at' igalia.com), Igalia.
+
+Contributors
+
+    Ricardo Garcia, Igalia
+
+Notice
+
+    Copyright (c) 2021-2024 The Khronos Group Inc. Copyright terms at
+        http://www.khronos.org/registry/speccopyright.html
+
+Status
+
+    Draft
+
+Version
+
+    Last Modified Date: 2024/10/21
+    Revision: 1
+
+Dependencies
+
+    This extension can be applied to OpenGL GLSL versions 1.30
+    (#version 130) and higher.
+
+    This extension can be applied to OpenGL ES ESSL versions 3.00
+    (#version 300) and higher.
+
+    This extension is written against the OpenGL Shading Language
+    Specification version 4.60, dated July 23, 2017.
+
+Overview
+
+    This extension removes the requirement that offset arguments to the
+    texture*Offset functions must be constant.
+
+Modifications to the OpenGL Shading Language Specification, Version 4.60
+
+    Including the following line in a shader can be used to control the
+    language features described in this extension:
+
+      #extension GL_KHR_non_const_texture_offset : <behavior>
+
+    where <behavior> is as specified in section 3.3.
+
+    New preprocessor #defines are added:
+
+      #define GL_KHR_non_const_texture_offset        1
+
+
+Modifications to Chapter 8 of the OpenGL Shading Language Specification
+(Built-in Functions)
+
+    Change to section 8.9.2, Texel Lookup Functions
+
+    In the table row corresponding to the textureOffset functions, remove the
```

Comment:
nit: use `texture*Offset` to include all variants.

Reply:
I can do that, no problem, but it's worth noting all other variants (texelFetchOffset, textureProjOffset, textureLodOffset, textureProjLodOffset, textureGradOffset and textureProjGradOffset) merely reference textureOffset, so the sentence to be removed only appears once.

Reply:
Ack, the current language is fine.